internal/containerupdate/docker_container_updater.go

```diff
@@ -40,16 +40,16 @@ func (cu *DockerUpdater) UpdateContainer(ctx context.Context, cInfo store.Contai
 		return errors.Wrap(err, "rmPathsFromContainer")
 	}
 
-	// TODO(maia): catch errors -- CopyToContainer doesn't return errors if e.g. it
-	// fails to write a file b/c of permissions =(
-	err = cu.dCli.CopyToContainerRoot(ctx, cInfo.ContainerID.String(), archiveToCopy)
+	err = cu.dCli.ExecInContainer(ctx, cInfo.ContainerID, model.Cmd{
+		Argv: []string{"tar", "-C", "/", "-x", "-f", "-"},
```

Comment:
this is the same invocation as in the exec updater, right? nbd but might be smart to make this a shared arg list so that we force the two invocations to stay in sync.

Reply:
also probably worth a comment why we're using exec+tar instead of the docker client's builtin Copy functionality

Reply:
added!